app/models/champs/drop_down_list_champ.rb

```diff
@@ -1,2 +1,7 @@
 class Champs::DropDownListChamp < Champ
+  THRESHOLD_NB_OPTIONS_AS_RADIO = 5
+
+  def render_as_radios?
+    drop_down_list.enabled_non_empty_options.size <= Champs::DropDownListChamp::THRESHOLD_NB_OPTIONS_AS_RADIO
```

Comment:
Trop bien – oui, ça m'a l'air mieux que de mettre de la logique dans le template.

Par contre on peut virer le namespace de la constante, non ?

```suggestion
    drop_down_list.enabled_non_empty_options.size <= THRESHOLD_NB_OPTIONS_AS_RADIO
```